src/controllers/user.controller.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const NOT_FOUND = 404;
+const BAD_REQUEST = 400;
+const CREATED = 201;
+const NO_CONTENT = 204;
+const OK = 200;
```

Comment:
And this place. 
Please add it to the variable in the global scope, and use it in all places. 
Thanks)